external/vcm/vcm/cubedsphere/coarsen_restarts.py

```diff
@@ -61,6 +78,7 @@ def coarsen_restarts_on_sigma(
     grid_spec: xr.Dataset,
     restarts: Mapping[str, xr.Dataset],
     coarsen_agrid_winds: bool = False,
+    mass_weighted: bool = True,
```

Comment:
I added it since I will run an experiment using the non-default area-weighting approach for the model level component (up to this point we have always used mass-weighting).  We could remove the parameter if/when we feel ready to commit to area-weighting or mass-weighting.

Reply:
Okay. I think in general we should minimize the configuration burden for this workflow, so if we can remove it at some point that would probably be good. But okay to leave in for now.